anagram.py

```diff
@@ -1,6 +1,5 @@
 import argparse
 
-
```

Comment:
PEP8 guidelines state that there should be 2 blank lines between the imports and the module code, so this blank line should stay in.